tools/debug/vbconvert/main.go

```diff
@@ -90,11 +74,7 @@ func run(args arguments) {
 		os.Exit(1)
 	}
 
-	conduitType := conduitVB{
-		Blk:   algodType.Blk,
-		Delta: modelDelta,
-	}
-	err = enc.Encode(conduitType)
+	err = enc.Encode(algodType)
```

Comment:
this file can be removed. it was used to convert ledgercore.StateDelta to model.LedgerStateDelta